week06/LCA_of_a_binary_tree.cpp

```diff
@@ -0,0 +1,11 @@
+// Time complexity: O(N) , N = Number of nodes.
+// Space complexity: O(1).
```

Comment:
Space complexity is O(H), where H = max depth of the tree. It is because of the stack memory used by the recursion.